starknet/src/utils/types.cairo

```diff
@@ -150,360 +184,327 @@ impl StorageAccessFinalizationStatus of StorageAccess<FinalizationStatus> {
         }
     }
 
-    fn write(
-        address_domain: u32, base: StorageBaseAddress, value: FinalizationStatus
+    fn write_at_offset_internal(
+        address_domain: u32, base: StorageBaseAddress, offset: u8, value: FinalizationStatus
     ) -> SyscallResult<()> {
-        StorageAccess::write(
+        StorageAccess::write_at_offset_internal(
             address_domain,
             storage_base_address_from_felt252(
                 storage_address_from_base_and_offset(base, 0_u8).into()
             ),
+            offset,
             FinalizationStatusIntoU8::into(value)
         )
     }
-}
 
-impl LegacyHashChoice of LegacyHash<Choice> {
-    fn hash(state: felt252, value: Choice) -> felt252 {
-        LegacyHash::hash(state, ChoiceIntoU8::into(value))
+    fn size_internal(value: FinalizationStatus) -> u8 {
+        1_u8
```

Comment:
this returns the number of `bytes` used by `value`, right? Just double checking in case it's supposed to be the number of `bits`

Reply:
no its the number of felts